### PR TITLE
Fixed the base url for the SmartSmsSolutions class

### PR DESCRIPTION
**Pull request by kellslte**
I noticed that text messages via the SmartSmsSolutions class were not sending so I dug into the code and noticed that the base url was incorrect. I changed that to the correct one and the messages started sending.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "Djunehor\\Sms\\Test\\": "tests"
+      "Djunehor\\Sms\\Tests\\": "tests"
```
Comment (djunehor) [inline composer.json]: This unnecessary change breaks the app. The correct namespace for tests is `Djunehor\Sms\Test\`

Comment (kellslte) [reply composer.json]: Ok. I have been having a little difficult setting up my testbench that's probable why I didn't catch that.

Comment (djunehor) [reply composer.json]: Kindly revert back. Once you do, you'll see that the scrutinizer tests will pass

Comment (kellslte) [reply composer.json]: alright

**Comment by Goodluck00112** [comment]
WOW. GREAT
